src/commands/synthetics/run-tests-lib.ts

```diff
@@ -52,37 +52,22 @@ export const executeTests = async (
   summary: Summary
 }> => {
   const api = getApiHelper(config)
-
-  const publicIdsFromCli = config.publicIds.map((id) => ({
-    config: {
-      ...config.global,
-      ...(config.locations?.length ? {locations: config.locations} : {}),
-    },
-    id,
-  }))
-  let testsToTrigger: TriggerConfig[]
   let tunnel: Tunnel | undefined
+  let triggerConfigs: TriggerConfig[] = []
 
   const stopTunnel = async () => {
     if (tunnel) {
       await tunnel.stop()
     }
   }
 
-  if (publicIdsFromCli.length) {
-    testsToTrigger = publicIdsFromCli
-  } else {
-    try {
-      testsToTrigger = await getTestsList(api, config, reporter, suites)
-    } catch (error) {
-      throw new CriticalError(
-        isForbiddenError(error) ? 'AUTHORIZATION_ERROR' : 'UNAVAILABLE_TEST_CONFIG',
-        error.message
-      )
-    }
+  try {
+    triggerConfigs = await getTriggerConfigs(api, config, reporter, suites)
+  } catch (error) {
+    throw new CriticalError(isForbiddenError(error) ? 'AUTHORIZATION_ERROR' : 'UNAVAILABLE_TEST_CONFIG', error.message)
```

Comment:
I love that this error handling is at the same level as the one for `NO_TESTS_TO_RUN` 🙌